attributes/default.rb

```diff
@@ -1,6 +1,6 @@
-node.default[:packer][:url_base] = "https://dl.bintray.com/mitchellh/packer"
-node.default[:packer][:version] = "0.7.1"
-node.default[:packer][:arch] = kernel['machine'] =~ /x86_64/ ? "amd64" : "386"
+node.default[:packer][:url_base] = 'https://releases.hashicorp.com/packer/0.11.0'
+node.default[:packer][:version] = 'packer_0.11.0'
+node.default[:packer][:arch] = node['kernel']['machine'] =~ /x86_64/ ? "amd64" : "386"
```

Comment:
ℹ️ It looks like this was simply updated from what it was before, but do we have a need for 32 bit archs?

Reply:
No.  These changes are just to get this to a functional state again so it can be used with hosts running Chef 13.  I expect most of this will change, or be abandoned entirely.